CLAUDE.md

```
@@ -0,0 +1,347 @@
+    expect(sidebarOpen).toBe(true);
+  });
+});
+
+// ❌ DON'T: Try to render Tamagui components
+// This will fail due to DOM API requirements
+render(<AppLayout><Text>Test</Text></AppLayout>);
+```
+
+#### 3. Custom Hooks (`hooks/`)
+
+**🎯 RECOMMENDED: Use @testing-library/react-hooks**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)